src/tests/happy-path.test.ts

```diff
@@ -12,6 +11,8 @@ import { appendEntity } from '../draft-js-utils'
 
 describe('happy path', () => {
   const mocks = createMocks()
+  const app = () => mocks.popupWindow.document.querySelector('#app-container > .app')!
```

Comment:
What are the advantages of making the `app` a function instead of an element?

Reply:
It can change as the app rerenders